edx_sga/sga.py

```diff
@@ -1015,6 +1072,82 @@ def runtime_user_is_staff(self):
         """
         return self.is_course_staff()
 
+    def team_grading_data(self):
+        """
+        Return team member assignment information for display on the
+        grading screen.
+        """
+        members = self.get_teams_members()
+        if not members:
+            return {"assignments":{}}
```

Comment:
Should this return an empty list instead of returning an empty dict?

Reply:
you are right